bigchaindb/fastquery.py

```diff
@@ -18,15 +18,14 @@ class FastQuery:
     a byzantine node.
     """
 
-    def __init__(self, connection, me):
+    def __init__(self, connection):
         self.connection = connection
-        self.me = me
 
     def filter_valid_block_ids(self, block_ids, include_undecided=False):
         """Given block ids, return only the ones that are valid."""
         block_ids = list(set(block_ids))
         votes = query.get_votes_for_blocks_by_voter(
```

Comment:
This function does not even exist anymore. What do you think about removing `filter_valid_block_ids` together with 2 tests from `test_fastquery.py`?